### PR TITLE
EMS and JIRA hours log

### PR DESCRIPTION
**Pull request by RashmiDholakiya**
API Implementation:
Compare and check EMS and JIRA hours , if JIRA hours are less then EMS hours then mark absent to particular employee.

### CODE REVIEW

File: app/Console/Commands/CheckJiraHours.php [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+	 * The name and signature of the console command.
+	 * @var string
+	 */
+	protected $signature = 'Ems:CheckJiraHours';
+
+	/**
+	 * The console command description.
+	 * @var string
+	 */
+	protected $description = 'EMS Post API';
```
Comment (poojajajal) [inline app/Console/Commands/CheckJiraHours.php]: Add proper description.

File: app/Console/Commands/CheckJiraHours.php [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+					//Find JIRA hours
+
+					$worklog           = Timelog::whereStaffId($staff->id)->whereStarted($date)->sum('time_spent');
+					$actual_jira_hours = gmdate('H:i:s', ($worklog));
+					$actual_ems_hours  = $ems->actual_hours;
+					//Comparing EMS and JIRA hours
+					if ($actual_jira_hours != NULL && $actual_jira_hours != '00:00:00' && $actual_ems_hours != NULL && $actual_ems_hours != '00:00:00') {
+						$diff = $actual_ems_hours - $actual_jira_hours;
+						//IF difference is greater then 1 hour, then update EMS
+						if ($diff > 1 && $diff < 4) {
```
Comment (poojajajal) [inline app/Console/Commands/CheckJiraHours.php]: Fix  indentation.

File: app/constant.php [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<?php
+
+define('STATUS_OK', 200);
+define('HOST_PATH', 'http://localhost/');
```
Comment (poojajajal) [inline app/constant.php]: This is wrong. You don't write localhost such a way. 
User like this define('URL_PROJECT_PATH', URL::to('/') . '/');

File: app/Console/Commands/CheckJiraHours.php [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+		parent::__construct();
+	}
+
+	/**
+	 * Execute the console command.
+	 * @return mixed
+	 */
+	public function handle()
+	{
+		//
```
Comment (poojajajal) [inline app/Console/Commands/CheckJiraHours.php]: Remove this comment !

File: app/Console/Commands/CheckJiraHours.php [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+		$client = new Client();
+		//Call to EMS API to get the employee details
+
+		$date = Carbon::yesterday()->format('Y-m-d');
+		$res  = $client->get(HOST_PATH . 'ems/ems_api/employee_list/' . $date);
```
Comment (poojajajal) [inline app/Console/Commands/CheckJiraHours.php]: You can also put "ems/ems_api" on constant file.

File: app/Console/Commands/CheckJiraHours.php (hunk 1 above)
Comment (RashmiDholakiya) [inline app/Console/Commands/CheckJiraHours.php]: Done

File: app/Console/Commands/CheckJiraHours.php (hunk 2 above)
Comment (RashmiDholakiya) [inline app/Console/Commands/CheckJiraHours.php]: Done

File: app/constant.php (hunk 3 above)
Comment (RashmiDholakiya) [inline app/constant.php]: Done

File: app/Console/Commands/CheckJiraHours.php (hunk 5 above)
Comment (RashmiDholakiya) [inline app/Console/Commands/CheckJiraHours.php]: Done

**Pull request closed without merging**